CatalogTests/catalog.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { Browser, Builder, By, WebDriver } from "selenium-webdriver";
+import * as chrome from "selenium-webdriver/chrome";
+import { path } from "chromedriver";
+import { BasePage } from "./helpers/driver-manager";
+import { URL } from "./helpers/urls";
+import * as pages from "./pages";
+import { CatalogPage } from "./pages/catalog-page";
+import { SearchPage } from "./pages/search-page";
+import { text } from "./helpers/texts";
+import { time } from "console";
+
+describe("Catalog Onliner", () => {
+    let driver: WebDriver;
+    beforeEach(async () => {
```

Comment:
### As I can see tests fail, because timeouts are incorrectly configured

U have 2 types of timeouts: `jest` and `webdriver (next WD)`


> As I can see u set WD timeouts here (line 18):
> 
>  `await driver.manage().setTimeouts({ implicit: 10000 });`
> 
> which waits only for an element locator to succeed, please read about WD timeous [here](https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_Options.html#setTimeouts)


> And also you should set `jest` timeouts especially for each jest func like `afterEach()` / `test()` etc. 
> 
> Please read about [jest timeouts](https://jestjs.io/docs/api#beforeeachfn-timeout) and refactor tests/hooks timeouts where it's necessary and how it's necessary (in our work we use 15sec as a default step timeout)
> 
> (Imagine that I'm running tests with a slow connection and I need about 10 seconds just to open main page)